src/frontend/src/components/SchemaForm.tsx

```diff
@@ -131,6 +139,16 @@ export default function SchemaForm({ onChange, schema }: Readonly<SchemaFormProp
                 />
               </div>
               <div className="controls">
+                <IconButton
```

Comment:
might be being picky -  but if there is only one field we don't need the arrows, maybe refactor so that the icons only appear when there is more than 1 field?

Reply:
fixed